### PR TITLE
Add the option to copy snapshots to a different region

### PR DESCRIPTION
**Pull request by komljen**
# Description

Add the option to copy snapshots to a different region.

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -42,6 +42,11 @@ resource "aws_redshift_cluster" "this" {
   preferred_maintenance_window        = var.preferred_maintenance_window
   allow_version_upgrade               = var.allow_version_upgrade
 
+  snapshot_copy {
```
Comment (antonbabenko) [inline main.tf]: AFAIK `snapshot_copy` block should be optional also if the user does not want to copy snapshots.

Please upgrade to use `dynamic` block for this.

Comment (komljen) [reply main.tf]: I think if the `destination_region` is not specified (default empty string), snapshots will not be copied. But, the dynamic block should be a way to go probably.

Comment (komljen) [reply main.tf]: Done.

**Comment by antonbabenko** [comment]
Thanks @komljen !

[v2.4.0](https://github.com/terraform-aws-modules/terraform-aws-redshift/releases/v2.4.0) has been just released.

**Pull request merged**